apps/users/api/internal/register.py

```
@@ -12,61 +14,69 @@
+                    "email": user.email,
+                    "name": user.name,
+                    "phone": str(user.phone) if user.phone else "",
+                    "is_active": user.is_active,
+                    "created": True,
+                    "is_profile_completed": developer_profile.profile_completed,
+                },
+            }
+        except Exception as e:
+            raise ItqanError(
+                error_name="registration_failed",
+                message=f"Registration failed: {str(e)}",
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)